mix.exs

```diff
@@ -65,7 +75,11 @@ defmodule Videoroom.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
-      setup: ["deps.get"]
+      setup: ["deps.get"],
+      integration_test: [
+        "cmd docker pull ghcr.io/jellyfish-dev/jellyfish:latest",
```

Comment:
Imo we shouldn't lock to the latest tag. Elixir SDK case is different as we always want it to be up to date with Jellyfish but in case of real producst, I can image they might not use the newest Jellyfish version

Also, do we need to pull the image manually? Wouldn't docker-compose do this for us?

Reply:
`docker compose up` doesn't pull the image automatically. But we can use `docker compose pull` to get the latest image specified in the docker-compose file.

For now we will use the `edge` tag, but in the future we might lock to a specific release tag of jellyfish.